test/package.js

```diff
@@ -77,14 +81,17 @@ test('Every rule is defined in readme.md usage and list of rules in alphabetical
 		}
 	} while (match);
 
-	for (const file of ruleFiles) {
-		const name = path.basename(file, '.js');
+	const availableRules = ruleFiles
+		.map(file => path.basename(file, '.js'))
+		.filter(name => !deprecatedRules.includes(name));
+
+	for (const name of availableRules) {
 		t.truthy(usageRules[`unicorn/${name}`], `'${name}' is not described in the readme.md ## Usage`);
 		t.truthy(rules.includes(name), `'${name}' is not described in the readme.md ## Rules`);
 	}
 
-	t.is(Object.keys(usageRules).length - ignoredRules.length, ruleFiles.length, 'There are more rules in readme.md ## Usage than rule files.');
-	t.is(Object.keys(rules).length, ruleFiles.length, 'There are more rules in readme.md ## Rules than rule files.');
+	t.is(Object.keys(usageRules).length - ignoredRules.length, availableRules.length, 'There are more rules in readme.md ## Usage than rule files.');
+	t.is(Object.keys(rules).length, availableRules.length, 'There are more rules in readme.md ## Rules than rule files.');
```

Comment:
Would it be worth it to add a check that ensures deprecated rules are described in the readme?

Reply:
I don't think it's really nessary.